app/javascript/components/Layout/index.jsx

```diff
@@ -57,7 +57,10 @@ const Layout = ({ children }) =>
     <>
         <GlobalStyle/>
         <Header/>
-        <PhaseBanner/>
+        {
+            bannerTag || ledeText || bannerTag &&
```

Comment:
```suggestion
            bannerTag || ledeText || bannerText &&
```

Reply:
What happens if you fill out the text fields but not the banner tag field?

Reply:
We have 3 inputs for the banner, The banner tag is a type of title, lede text is bold and the text is the last part of the banner. I thought if we have at least one of them we could display a banner. If we have the text but not the banner we still can have the banner display but without the tag.

Reply:
I have a feeling that we need brackets around the first part of that statement `(bannerTag || ledeText || bannerText) && ...` for that to work, otherwise if the last one (`bannerText` - or `bannerTag` as it currently is 😄 ) is empty it might not render correctly. But I might be wrong!